controllers/apply/standard-proposal/fields.js

```diff
@@ -207,7 +207,10 @@ module.exports = function fieldsFor({ locale, data = {} }) {
 
             return Joi.when(Joi.ref('projectCountries'), {
                 is: isEnglandSelected,
-                then: [validAllEngland, validRegionOptions],
+                then: Joi.alternatives([
```

Comment:
This is the resulting fix. The `[]` syntax maps to `Joi.alternatives`. Needed an explicit `.required()` otherwise the absence of the field is considered OK.